src/backend/test/unit/pia-intake/pia-intake.service.spec.ts

```diff
@@ -1980,6 +1905,59 @@ describe('PiaIntakeService', () => {
       };
       expect(result).toEqual(expectedResult);
     });
+
+    // Scenario 20: User is a drafter, has no roles, and is filtering for COMPLETE PIAs
+    it('succeeds when the user is only a drafter and is filtering for COMPLETE PIAs', async () => {
+      const user: KeycloakUser = { ...keycloakUserMock };
+      const userRoles = [];
+      const piaIntakeEntity = { ...piaIntakeEntityMock };
+      const query: PiaIntakeFindQuery = {
+        page: 1,
+        pageSize: 12,
+        filterByStatus: PiaIntakeStatusEnum.COMPLETE,
+      };
+
+      piaIntakeRepository.findAndCount = jest.fn(async () => {
+        delay(10);
+        return [[piaIntakeEntity], 100];
+      });
+
+      omitBaseKeysSpy.mockReturnValue({ ...getPiaIntakeROMock });
+
+      const result = await service.findAll(user, userRoles, query);
+
+      expect(piaIntakeRepository.findAndCount).toHaveBeenCalledWith({
+        where: [
+          {
+            isActive: true,
+            createdByGuid: user.idir_user_guid,
+            status: PiaIntakeStatusEnum.COMPLETE,
+          },
+          {
+            isActive: true,
+            invitee: {
+              createdByGuid: user.idir_user_guid,
+            },
+            status: PiaIntakeStatusEnum.COMPLETE,
+          },
+        ],
+        order: {
+          createdAt: -1,
+        },
+        skip: 0,
+        take: 12,
+      });
+
+      expect(omitBaseKeysSpy).toHaveBeenCalledTimes(1);
+
+      const expectedResult: PaginatedRO<GetPiaIntakeRO> = {
+        data: [getPiaIntakeROMock],
+        page: 1,
+        pageSize: 12,
+        total: 100,
+      };
+      expect(result).toEqual(expectedResult);
+    });
```

Comment:
This looks great! Just that if you can create a scenario each for when a user is an MPO, and when they are a CPO?